test_outputs/connlist/exposure_acs-security-demos_connlist_output.csv

```diff
@@ -0,0 +1,24 @@
+src,dst,conn
+backend/checkout[Deployment],backend/notification[Deployment],TCP 8080
+backend/checkout[Deployment],backend/recommendation[Deployment],TCP 8080
+backend/checkout[Deployment],payments/gateway[Deployment],TCP 8080
+backend/recommendation[Deployment],backend/catalog[Deployment],TCP 8080
+backend/reports[Deployment],backend/catalog[Deployment],TCP 8080
+backend/reports[Deployment],backend/recommendation[Deployment],TCP 8080
+frontend/webapp[Deployment],backend/checkout[Deployment],TCP 8080
+frontend/webapp[Deployment],backend/recommendation[Deployment],TCP 8080
+frontend/webapp[Deployment],backend/reports[Deployment],TCP 8080
+frontend/webapp[Deployment],backend/shipping[Deployment],TCP 8080
+payments/gateway[Deployment],payments/mastercard-processor[Deployment],TCP 8080
+payments/gateway[Deployment],payments/visa-processor[Deployment],TCP 8080
+{ingress-controller},frontend/asset-cache[Deployment],TCP 8080
+{ingress-controller},frontend/webapp[Deployment],TCP 8080
+Exposure Analysis Result:,,
+src,dst,conn
+backend/checkout[Deployment],entire-cluster,UDP 5353
+backend/recommendation[Deployment],entire-cluster,UDP 5353
+backend/reports[Deployment],entire-cluster,UDP 5353
+entire-cluster,frontend/asset-cache[Deployment],TCP 8080
+entire-cluster,frontend/webapp[Deployment],TCP 8080
+frontend/webapp[Deployment],entire-cluster,UDP 5353
+payments/gateway[Deployment],entire-cluster,UDP 5353
```

Comment:
This is a bit inconsistent with `txt` output , where the first item in each line is the actual workload.. 
One option here is to have 2 sections instead of one - ingress exposure and egress exposure.. (that would require also header updates) 
What do you think is better?
@zivnevo what do you think?

Reply:
I don't see much value in reading exposure analysis results in CSV format.
Humans should better read the `txt` or the `md` version, while machines should read the `json` version.
So, any decision here is OK.

Reply:
OK, and what about `md` ? (should split to ingress / egress sections, to be consistent with `txt`?)

Reply:
yes, better be consistent between `md` and `txt`

Reply:
done, csv, json and md are consistent with txt now. 
i.e. exposure conns are separated into two sections : egress exposure and ingress exposure.